src/Bridge/Symfony/DependencyInjection/SonataFormExtension.php

```diff
@@ -48,6 +48,7 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader = new PhpFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('date.php');
         $loader->load('form_types.php');
+        // NEXT_MAJOR: Remove next line.
```

Comment:
You're on 2.x which is already the next major, so we can remove it no ?

Reply:
This code is used on the admin bundle, on twig files, if we want to be compatible with 4.x of the admin bundle we might want to remove later, otherwise we will need to check somehow the form extensions installed version on the admin to avoid using the removed filter (it also cant be deprecated on 1.x since its used there)

Reply:
Do you think it is better to do some checks on admin-bundle to detect if the twig function/filter exist?

Reply:
I don't think we'll release a lot of major so if we can remove it now it's better.
Depends on how hard is the check on the adminbundle side.

Reply:
Lets go for now with this and I will see if I can remove it prior to the release.